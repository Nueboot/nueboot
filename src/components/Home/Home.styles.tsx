import styled from "styled-components";

export const Headline = styled.div`
  margin: 120px 0 40px 0;
  height: calc(100vh - 260px);

  h1 {
    margin: 0;
    line-height: 1.2;
  }

  p {
    margin-top: 24px;
    color: #888;
    font-size: 20px;
    width: 60%;
    line-height: 1.6;
  }

  @media (max-width: 960px) {
    p {
      width: 100%;
    }
  }
`;
