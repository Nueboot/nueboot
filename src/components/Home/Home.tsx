import * as React from "react";
import { Headline } from "./Home.styles";

export default function Home() {
  return (
    <Headline>
      <h1>Community driven boot reviews.</h1>
      <h1>Unbiased and unfiltered.</h1>
      <p>
        Nueboot is a community driven boot resource for people all over the
        world. To be truly succesful we need contributors like you to write
        reviews and update boots to their correct statistics.
      </p>
    </Headline>
  );
}
