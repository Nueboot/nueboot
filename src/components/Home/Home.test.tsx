import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import Home from '../Home';

describe('Home', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<Home />);
  });

  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
