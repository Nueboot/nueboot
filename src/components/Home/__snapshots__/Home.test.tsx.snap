// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders as expected 1`] = `
<h1>
  Hi
</h1>
`;
